### DISCUSSION TITLE
Delete command messages once recieved.

### DISCUSSION THREAD

**Discussion by StrikerTheHedgefox** [Adikso/BashBot#17] [Ideas]
Suggesting that when you send the bot a command with a prefix, it deletes the original message, since the resulting command will show in the terminal session to begin with. This would keep messages from piling up in the channel you're operating the terminal in, keeping it in plain view without you needing to constantly issue "$.here".

**Comment by Adikso** [comment]
I added 'terminal.delete_messages' and 'terminal.interactive.delete_messages' options which are deleting message on receive.
